src/db/OrganizationTypes.ts

```diff
@@ -0,0 +1,118 @@
+import mongoose from 'mongoose'
+import { MUUID } from 'uuid-mongodb'
+
+import { ChangeRecordMetadataType } from './ChangeLogType.js'
+
+/**
+ * Organizations are OpenBeta accounts that are owned by organizations
+ * instead of people. For instance, the Boulder Climbing Community might own
+ * one such account. Organizations may have one or more 'org_admins' who
+ * manage the account. They may update organizations' profiles and perform
+ * special actions under the auspices of the organization. For example, org_admins
+ * of local climbing organizations (LCO) may put up bird nesting notices on
+ * the area pages associated with the LCO.
+ * */
+export type OrganizationType = {
+  _id: mongoose.Types.ObjectId
+  /**
+  * All external IDs for areas are expressed as UUIDs. As such, when resolving ids at the
+  * GQL layer use these values for querying and identification of organizations.
+  */
+  orgId: MUUID
+  /**
+   * Name of organization to be displayed on the site.
+   */
+  displayName: string
+  /**
+   * Type of organization. Currently we only support local climbing organizations, which
+   * are associated with certain climbing areas. In future there may be advocacy groups
+   * like the Access Fund or interest groups like the American Alpine Club that are not
+   * associated with any specific climbing areas.
+   */
+  orgType: OrgType
+  /**
+   * Areas associated with this organization. This has a strong relation to the
+   * areas collection, and contains only direct child areas - rather than all descendents.
+   */
+  associatedAreaIds: MUUID[]
+  /**
+   * Areas the organization explicitly does not want to be associated with.
+   * Takes precedence over associatedAreas. Intended use is for organizations to associate
+   * with large swathes of areas, and then exclude a few specific ones that are contentious
+   * or legally problematic. In other words, excludedAreas *unlike associatedAreas* have to be
+   * specified individually -- an area is not automatically excluded just because an ancestor is.
+   */
+  excludedAreaIds: MUUID[]
+  /**
+   * User-composed content that makes up most of the user-readable data in the system.
+   * See the IOrganizationContent documentation for more information.
+   **/
+  content: IOrganizationContent
+  /**
+   * If this organization has been edited, this field will contain the metadata about the
+   * last edit that was made.
+   */
+  _change?: ChangeRecordMetadataType
+  /** Used to delete an area.  See https://www.mongodb.com/docs/manual/core/index-ttl/ */
+  _deleting?: Date
+  createdAt?: Date
```

Comment:
Why would this be an optional field? The time of creation is always knowable at the time of creation

Reply:
Good point! I could enforce that these be mandatory, and have tests pass with minimal fixes. Made the change!